package.json

```diff
@@ -44,7 +44,7 @@
     "escape-html": "^1.0.3",
     "fresh": "~0.5.2",
     "http-assert": "^1.3.0",
-    "http-errors": "^1.6.3",
```

Comment:
^1.6.3 contains ^1.8.0, so there is no need to upgrade every dependency to the latest version. :)

Reply:
For this reason, I separated each dependency inside PR(s).